_data/Muskan-Mittal/Muskan-Mittal.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Muskan Mittal
+institution: IIT Roorkee 🚩 
+profile_pic: muskan.png
```

Comment:
Hey @Muskan-Mittal  , the profile_pic link should be exact as per the filename ( with extension ) . So `png` -> `jpg`

Reply:
Hey, I have updated it, please have a look.